xc7/utils/vivado_create_runme.py

```diff
@@ -11,7 +11,6 @@ def create_runme(f_out, args):
 
 read_verilog {bit_v}
 synth_design -top {top}
-write_checkpoint -force design_{name}_pre_route.dcp
```

Comment:
This was put here for a reason.  Something sourcing the tcl generates an error when either setting LOC/BEL and/or FIXED_ROUTE.

Reply:
Ok, I'll add another dcp creation than. Mainly because, for large designs, sourcing the tcl in Vivado takes quite a high amount of time, and it would be nice to have a dcp generated, in case P&R fails.